### ISSUE TITLE
Settings table updates for large values reverted by database repair

### ISSUE THREAD

**Issue by unka65** [Cacti/cacti#5561]
I noticed the following in thold database.php function thold_setup_database() that it changes the base application table settings, column "value" size to 4096.
```
/* increase the size of the settings table */
db_execute("ALTER TABLE settings
	MODIFY COLUMN `value`
	varchar(4096) NOT NULL default ''");
```	
I wonder if that is for values thold_down_text and thold_up_text.

In running the base application database repair for troubleshooting other, one repair was the settings table column value size from 4096 to 2048.
```
Checking Table: 'settings'
ERROR Col: 'value', Attribute 'Type' invalid. Should be: 'varchar(2048)', Is: 'varchar(4096)'

```
I thought I would to mention it in case it could be a potential data loss that should be noted somewhere if not already.

**Comment by TheWitness** [comment]
I'll be darn.  Let me transfer to cacti.

**Comment by TheWitness** [comment]
This will be resolved in the 1.2.26 release.
